vendor/vendor.json

```diff
@@ -861,6 +861,12 @@
 			"revision": "d1caa6c97c9fc1cc9e83bbe34d0603f9ff0ce8bd",
 			"revisionTime": "2016-07-20T23:31:40Z"
 		},
+		{
+			"checksumSHA1": "/JX5JopEWDtQKvyLopaD2/1YJXM=",
+			"path": "github.com/huawei-clouds/golangsdk/openstack/kms/v1/keys",
```

Comment:
it seems it's a repeated references with line 902. suggest to not use this repository as it deprecated. please check is it correct.

Reply:
Done